Fixes yesod-persistent test build failure:

[2 of 2] Compiling Main             ( test/Spec.hs, dist/build/test/test-tmp/Main.o )
Linking dist/build/test/test ...
/usr/lib/gcc/x86_64-pc-linux-gnu/4.8.1/../../../../x86_64-pc-linux-gnu/bin/ld: /usr/lib64/persistent-sqlite-1.2.1/ghc-7.6.3/libHSpersistent-sqlite-1.2.1.a(sqlite3.o): undefined reference to s
ymbol 'pthread_mutexattr_settype@@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-pc-linux-gnu/4.8.1/../../../../x86_64-pc-linux-gnu/bin/ld: note: 'pthread_mutexattr_settype@@GLIBC_2.2.5' is defined in DSO /lib64/libpthread.so.0 so try adding it to the 
linker command line
/lib64/libpthread.so.0: could not read symbols: Invalid operation
collect2: error: ld returned 1 exit status

diff --git a/persistent-sqlite.cabal b/persistent-sqlite.cabal
index c40a1d9..aaf0b1f 100644
--- a/persistent-sqlite.cabal
+++ b/persistent-sqlite.cabal
@@ -34,6 +34,7 @@ library
         extra-libraries: sqlite3
     else
         c-sources:   cbits/sqlite3.c
+        extra-libraries: pthread
 
 source-repository head
   type:     git
